client/src/components/ContactApplicantButton.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import {
+  Form,
+  Button,
+  Modal,
+  TextArea,
+  Icon,
+  Header,
+  Grid
+} from "semantic-ui-react";
+import { sendMessage } from "../api/messages";
+
+const ContactApplicantButton = props => {
+  const [open, setOpen] = React.useState(false);
+  const [messageSent, setMessageSent] = React.useState(null);
+  const user = JSON.parse(localStorage.getItem("user"));
+  const recruiterEmailAddress = user.email;
+  const recruiterId = user.id;
+  const [message, setMessage] = React.useState({
+    profile_public_id: `${props.profilePublicId}`,
+    message: `
+    Hi, 
+    I wanted to get in touch with you as at COMPANYNAME we are currently looking to hire some engineers that 
+    match your profile.
+    If you would be interested in speaking to me some more please reply to this email and we can arrange a time to speak. 
+    I look forward to speaking to you please get in touch on my email address below.
+    ${recruiterEmailAddress}`,
+    recruiter_id: `${recruiterId}`
```

Comment:
This also same as top comment, it should be only like this: `recruiter_id: recruiterId`

Reply:
Changed. Thanks.